<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BootstrapFiberTrackingForm</class>
 <widget class="QWidget" name="BootstrapFiberTrackingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelBootstrapDirectory">
     <property name="text">
      <string>Bootstrap Directory</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonBootstrapDirectory">
     <property name="text">
      <string>Open Directory...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelBootstrapsFound">
       <property name="text">
        <string>Bootstraps Found:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNrBootstrapsFound">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelNrBootstraps">
     <property name="text">
      <string>Number of Bootstraps</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="sliderNrBootstraps">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNrBootstrapsCounter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelDatasets">
     <property name="text">
      <string>Datasets</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboDatasets"/>
   </item>
   <item>
    <widget class="QLabel" name="labelAnisotropyDataset">
     <property name="text">
      <string>Anisotropy Datasets</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboAnisotropyDatasets"/>
   </item>
   <item>
    <widget class="QLabel" name="labelAnisotropy">
     <property name="text">
      <string>Anisotropy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSlider" name="sliderAnisotropy">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAnisotropyCounter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>30</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelSeedRegions">
     <property name="text">
      <string>Seed Regions</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="comboSeedRegions"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="buttonLoad">
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRun">
     <property name="text">
      <string>Run Fiber Tracking</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>222</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderAnisotropy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelAnisotropyCounter</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderNrBootstraps</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelNrBootstrapsCounter</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
